src/components/structures/RightPanel.js

```diff
@@ -45,11 +45,7 @@ class HeaderButton extends React.Component {
         const TintableSvg = sdk.getComponent("elements.TintableSvg");
         const AccessibleButton = sdk.getComponent("elements.AccessibleButton");
 
-        return <AccessibleButton
-            className="mx_RightPanel_headerButton"
-            key={this.props.key}
-            onClick={ this.onClick }
-        >
+        return <AccessibleButton className="mx_RightPanel_headerButton" onClick={ this.onClick } >
```

Comment:
trailing space before `>` :(

Reply:
plus I feel like you at one point told me props get no space padding within `{}`

Reply:
this is true, I've added an eslint rule for this - https://github.com/matrix-org/matrix-react-sdk/pull/1436

Reply:
:D